cmd/register/oidcconfig/cmd.go

```diff
@@ -120,7 +120,7 @@ func run(cmd *cobra.Command, argv []string) {
 
 	if !cmd.Flags().Changed(InstallerRoleArnFlag) && (interactive.Enabled() || confirm.Yes()) {
 		args.installerRoleArn = interactiveRoles.
-			GetInstallerRoleArn(r, cmd, args.installerRoleArn, MinorVersionForGetSecret)
+			GetInstallerRoleArn(r, cmd, args.installerRoleArn, MinorVersionForGetSecret, r.AWSClient.FindRoleARNs)
```

Comment:
@gdbranco Same here, it can be either classic or HCP for the purpose of the OIDC config? as the description says `STS Role ARN with get secrets permission`.

Reply:
Yeah, same idea. The only requirement is to have the perms. It doesn't really need to be the installer, it's just that we know the installer works for sure